src/Core/Framework/MessageQueue/ScheduledTask/Exception/ScheduledTaskNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\MessageQueue\ScheduledTask\Exception;
+
+use Shopware\Core\Framework\ShopwareHttpException;
+
+class ScheduledTaskNotFoundException extends ShopwareHttpException
+{
+    public function __construct(string $scheduledTaskId)
+    {
+        parent::__construct(sprintf('No scheduled task found for id %s', $scheduledTaskId), [
```

Comment:
we have some kind of templating within the ShopwareHttpException
```suggestion
        parent::__construct('No scheduled task found with ID "{{ scheduledTaskId }}"', [
```
just use the array key of the second parameters you give